vkubelet/vkubelet.go

```diff
@@ -38,10 +39,8 @@ type Server struct {
 
 // New creates a new virtual-kubelet server.
 func New(nodeName, operatingSystem, namespace, kubeConfig, taint, provider, providerConfig string) (*Server, error) {
-	var (
-		config *rest.Config
-		err    error
-	)
+	var config *rest.Config
```

Comment:
Fixing a few golint warnings. This definition of err was not used. Also restricted a few really long lines to 120 columns.